pkg/apicapi/apic_sync.go

```diff
@@ -137,6 +137,113 @@ func (conn *ApicConnection) apicObjCmp(current ApicObject,
 	return
 }
 
+func (conn *ApicConnection) setDeleteStatus(deleteBody *ApicObject, deldn string) {
+	if deleteBody == nil {
+		return
+	}
+	for _, val := range *deleteBody {
+		dn, okdn := val.Attributes["dn"]
+		if okdn && dn == deldn {
+			attrs := make(map[string]interface{})
+			attrs["dn"] = val.Attributes["dn"]
+			attrs["status"] = "deleted"
+			val.Attributes = attrs
+			conn.log.Debug("deleteBody: ", *deleteBody)
```

Comment:
Assuming all the logs will be cleaned up with another pass

Reply:
Yes. I will remove all the debug logs that is not necessary.